index.bs

```diff
@@ -189,66 +195,76 @@ For multi-layered images, the LayerSelectorProperty may be present as defined in
 
 NOTE: As defined in [[!MIAF]], a file that is primarily an image sequence still has at least a primary image item. Hence, it can also declare brands for signaling the image item.
 
+  <h2 id="file-constraints">File constraints</h3>
+
+  The following constraints are common to files compliant to this specification:
+  - The file shall be compliant with the [[!MIAF]] specification and list 'miaf' in the [=compatible_brands=] field of the [=FileTypeBox=].
+  - The file shall list <code>'[=avif=]'</code> in the [=compatible_brands=] field of the [=FileTypeBox=].
+  - The [=primary image=], or one of its alternates, shall be an [=AV1 Image Item=] or be a derived image that references one or more items that all are [=AV1 Image Items=].
+
   <h2 id="profiles">Profiles</h2>
 
   <h3 id="profiles-overview">Overview</h3>
 
   The profiles defined in this section are for enabling interoperability between [=AV1 Image File Format=] files and [=AV1 Image File Format=] readers/parsers. A profile imposes a set of specific restrictions and is signaled by brands defined in this specification.
 
-  The FileTypeBox should declare at least one profile that enables decoding of the primary image item, or one of its alternates. It is not an error for the encoder to include an auxiliary image that is not allowed by the specified profile(s).
-
-  If an image sequence brand is declared in the FileTypeBox, if a profile is declared in the FileTypeBox, the profile shall also enable decoding of at least one image sequence track. The profile should allow decoding of any associated auxiliary image sequence tracks, unless it is acceptable to decode the image sequence without its auxiliary image sequence tracks.
+  The [=FileTypeBox=] should declare at least one profile that enables decoding of the primary image item, or one of its alternates. It is not an error for the encoder to include an auxiliary image that is not allowed by the specified profile(s).
 
-  <h3 id="profiles-constraints">Common constraints</h3>
-
-  The following constraints are common to all profiles defined in this specification:
-  - The file shall be compliant with the [[!MIAF]] specification and list 'miaf' in the [=compatible_brands=] field of the [=FileTypeBox=].
-  - The [=primary image=], or one of its alternates, shall be an [=AV1 Image Item=] or be a derived image that references one or more items that all are [=AV1 Image Items=].
+  If an image sequence brand is declared in the [=FileTypeBox=], if a profile is declared in the [=FileTypeBox=], the profile shall also enable decoding of at least one image sequence track. The profile should allow decoding of any associated auxiliary image sequence tracks, unless it is acceptable to decode the image sequence without its auxiliary image sequence tracks.
```

Comment:
Nit: In the first sentence, I think the second "if" could be replaced with "and".